

class UsernameTakenException(Exception):
    pass


class EmailTakenException(Exception):
    pass
